samples/model-builder/endpoint_predict_sample.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from google.cloud import aiplatform
+
+#  [START aiplatform_sdk_endpoint_predict_sample]
+def endpoint_predict_sample(project : str, 
+                            location : str, 
+                            instances : str, 
```

Comment:
The type for instances should be a list.  Do we plan to have specialized prediction samples that show how to structure the instances at least for AutoML models?

Reply:
fixed